src/plugins/orangehrmDashboardPlugin/test/fixtures/QuickLaunchAPITest3.yml

```diff
@@ -20,7 +20,11 @@ UserRole:
 User:
   - {"id": 1,"user_role_id": 1,"emp_number": 1,"user_name": "admin","user_password": "$2y$12$cRXToxVun41SeRgsWTSzP.2cJmj4AZWi/0.BZr529V8p68PCSnRQ."}
   - {"id": 2,"user_role_id": 2,"emp_number": 2,"user_name": "sharuka","user_password": "$2y$12$EH.mp1hqHt7BW4EpaeOCKeQTyokMLYWVpMSitHtoMu/W3K2xOf5p."}
-  - {"id": 3,"user_role_id": 2,"emp_number": 3,"user_name": "rajitha","user_password": "$2y$12$f2zH9NNS.GrX/Dmx8r2eWuJt986aPLOUF8LvEj.8aAvFxEmaPIoNa"}
+  - {"id": 3,"user_role_id": 3,"emp_number": 3,"user_name": "rajitha","user_password": "$2y$12$f2zH9NNS.GrX/Dmx8r2eWuJt986aPLOUF8LvEj.8aAvFxEmaPIoNa"}
```

Comment:
Btw, I don't like to use my name.
use https://github.com/orangehrm/orangehrm/blob/main/src/plugins/orangehrmLeavePlugin/test/fixtures/EmployeeLeaveEntitlementUsageLeaveReportDataAPITest.yaml#L59-L65 kind of dummy data :relaxed: